front/app/containers/Admin/projects/edit/general/index.tsx

```diff
@@ -1,576 +0,0 @@
-    });
-
-    return !hasErrors;
-  };
-
-  save = async (
-    participationContextConfig: IParticipationContextConfig | null = null
-  ) => {
-    await save.apply(this, [participationContextConfig]);
-  };
-
-  validateProjectSlug = (slug: string) => {
-    const isSlugValid = validateSlug(slug);
-
-    this.setState({
-      showSlugErrorMessage: !isSlugValid,
-      submitState: isSlugValid ? 'enabled' : 'disabled',
-    });
-  };
-
-  handleFieldUpdate = (fieldUpdates: { [fieldPath: string]: any }) => {
-    this.setState((prevState) => {
-      const newState = { ...prevState };
-      set(newState, 'submitState', 'enabled');
-      for (const fieldPath in fieldUpdates) {
-        set(newState, fieldPath, fieldUpdates[fieldPath]);
-      }
-      return deepMerge(prevState, newState);
-    });
-  };
-
-  render() {
-    const {
-      publicationStatus,
-      projectType,
-      titleError,
-      project,
-      projectHeaderImage,
-      projectImages,
-      projectFiles,
-      processing,
-      projectAttributesDiff,
-      areasOptions,
-      areaType,
-      submitState,
-      apiErrors,
-      slug,
-      showSlugErrorMessage,
-      currentTenant,
-    } = this.state;
-
-    const { authUser } = this.props;
-
-    if (
-      !isNilOrError(authUser) &&
-      (!get(this.props, 'params.projectId') ||
-        (get(this.props, 'params.projectId') && project !== undefined))
-    ) {
-      const projectAttrs = {
-        ...(project ? project.data.attributes : {}),
-        ...projectAttributesDiff,
-      } as IUpdatedProjectProperties;
-      const areaIds =
-        projectAttrs.area_ids ||
-        (project &&
-          project.data.relationships.areas.data.map((area) => area.id)) ||
-        [];
-      const areasValues = areaIds
-        .filter((id) => {
-          return areasOptions.some((areaOption) => areaOption.value === id);
-        })
-        .map((id) => {
-          return areasOptions.find(
-            (areaOption) => areaOption.value === id
-          ) as IOption;
-        });
-
-      const selectedTopicIds = getSelectedTopicIds(
-        projectAttributesDiff,
-        project
-      );
-
-      return (
-        <StyledForm
-          className="e2e-project-general-form"
-          onSubmit={this.onSubmit}
-        >
-          <Section>
-            {get(this.props, 'params.projectId') && (
-              <>
-                <SectionTitle>
-                  <FormattedMessage {...messages.titleGeneral} />
-                </SectionTitle>
-                <SectionDescription>
-                  <FormattedMessage {...messages.subtitleGeneral} />
-                </SectionDescription>
-              </>
-            )}
-
-            <ProjectStatusPicker
-              publicationStatus={publicationStatus}
-              handleStatusChange={this.handleStatusChange}
-            />
-
-            <ProjectNameInput
-              projectAttrs={projectAttrs}
```

Comment:
Only passed the titleMultiloc here instead of all projectAttrs. Reason has to do with the validation and typing I believe. Not sure anymore, but I didn't do it just for fun. :D